src/index.ts

```diff
@@ -141,12 +146,20 @@ program
         "-c, --category [category]",
         "The Category name that best fits this block"
     )
+    .option(
+        "-i, --integration [integration]",
+        `The integration this block is built for. Options are: ${Object.keys(
+            INTEGRATIONS
+        )
+            .map((x) => `"${x}"`)
+            .join(", ")}. Defaults to 'standard'.`
```

Comment:
**Suggestion:** extract the integration-listing code to a named function:
```typescript
`The integration this block is built for. Options are: ${listIntegrations()}. Defaults to 'standard.'`
```

Reply:
Couldn't find a good place for a function to live, but I did extract the value and name it, which helps with clutter!